### ISSUE TITLE
rnnt loss with fastemit

will rnnt loss support fastemit？

### ISSUE THREAD

**Issue by Mddct** [pytorch/audio#2593]

**Comment by carolineechen** [comment]
Hi @Mddct, we are planning to add support for fastemit in our next release.

**Comment by carolineechen** [comment]
Hi @Mddct, just to follow up on this, we were unable to get it out in time for release 0.13 which launched today (our current loss implementation fuses the log softmax which makes fast emit more complicated to implement), but we are still looking to add it (along with other RNNT loss improvements) in release 0.14. I'll keep this thread updated!

**Comment by Mddct** [comment]
any update on this?
